frontend/src/libs/components/app/app.tsx

```diff
@@ -16,6 +33,14 @@ const App: React.FC = () => {
     }
   }, [isRoot, dispatch]);
 
+  useEffect(() => {
+    void dispatch(authActions.getAuthenticatedUser());
+  }, [dispatch]);
+
+  if (authenticatedUserDataStatus !== DataStatus.FULFILLED) {
+    return <Loader />;
+  }
```

Comment:
I suggest we show the loader in Pending state, because currently the Loader will be displayed forever if the request gets rejected

Reply:
Here we would prefer to leave it as it is